src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/213396167-36d56606-dc05-41c7-a440-d426191ee452.png)

Reply:
Not fixed